soot-infoflow-android/src/soot/jimple/infoflow/android/manifest/ProcessManifest.java

```diff
@@ -629,4 +682,25 @@ else if (intentFilter.getTag().equals("category") && intentFilter.getAttribute("
 		return allLaunchableActivities;
 	}
 
-}
\ No newline at end of file
+	public void test() {
```

Comment:
What is this method for?

Reply:
Hello Steven, 
It was a test I forgot to delete for another feature I'm working on.
Let me fix that